src/components/OSCALLoader.js

```diff
@@ -6,6 +6,12 @@ import OSCALCatalog from "./OSCALCatalog";
 import OSCALComponentDefinition from "./OSCALComponentDefinition";
 import OSCALProfile from "./OSCALProfile";
 import OSCALLoaderForm from "./OSCALLoaderForm";
+import {
+  defaultOscalCatalogUrl,
+  defaultOSCALComponentUrl,
+  defaultOSCALProfileUrl,
+  defaultOscalSspUrl,
+} from "../test-data/Urls";
```

Comment:
Do we want the `OSCALLoader` to depend on `/test-data`?

Reply:
Those imported Urls are simply the default Urls, which were still in their respective files, so I decided to move them to the directory containing the test data. If it makes more sense, I can just move those Urls back or create a separate directory/file for those Urls.

Reply:
It is only used in the `OSCALLoader` from what I can tell. Is there a reason to move them?

Reply:
I moved them because I assumed we were migrating all data and urls to the test-data directory, as a standard.

Reply:
I'd like to find a way to not import `test-data` in actual code. Our production code should not rely on tests or their data. Can we investigate another path forward here? Maybe we move these values into app code and the tests import from the app code?

Reply:
I vote to leave them in the OSCALLoader.js file itself. I think it'd be easier to read and develop if those are used as constants in that piece of code and not lumped in with the test-data urls.

Reply:
Ok, seems like it's best to move them back to the OSCALLoader.js file, so I'll go ahead and do that.

Reply:
Yeah I agree, minimizing the coupling of data is important if we want the components to be usable separately.